homeassistant/components/deconz/select.py

```diff
@@ -55,6 +67,39 @@ def async_add_presence_sensor(_: EventType, sensor_id: str) -> None:
     )
 
 
+class DeconzAirPurifierFanMode(DeconzDevice[AirPurifier], SelectEntity):
+    """Representation of a deCONZ air purifier fan mode entity."""
+
+    _name_suffix = "Fan Mode"
+    unique_id_suffix = "fan_mode"
+    _update_key = "mode"
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_options = [
+        AirPurifierFanMode.OFF.value,
```

Comment:
I'm missing translation strings for the options.

Reply:
Oh good point, I didn't look at it because it didn't have entity name = true but that's unrelated

Reply:
I will try to get it done before beta